src/components/Footer.tsx

```diff
@@ -38,43 +38,45 @@ const footerLogo = classnames(
   flexDirection('flex-row'),
   space('space-x-4')
 )
+const footerContainer = classnames(inset('bottom-0', 'left-0', 'right-0'))
 
 export default function () {
   const { md } = useBreakpoints()
 
   return (
     <>
-      <Spacer />
-      <div className={socialContainerCard}>
-        <div className={linkContainer}>
-          <FooterLink url="https://blog.bigwhalelabs.com/">
-            <div className={footerLogo}>
-              <FooterLogo />
-              <span>Blog</span>
-            </div>
-          </FooterLink>
-          <Delimiter />
-          <FooterLink internal url="/terms">
-            Terms of service
-          </FooterLink>
-          <Delimiter />
-          <FooterLink internal url="/privacy">
-            Privacy policy
-          </FooterLink>
+      <div className={footerContainer}>
+        <div className={socialContainerCard}>
+          <div className={linkContainer}>
+            <FooterLink url="https://blog.bigwhalelabs.com/">
+              <div className={footerLogo}>
+                <FooterLogo />
+                <span>Blog</span>
+              </div>
+            </FooterLink>
+            <Delimiter />
+            <FooterLink internal url="/terms">
+              Terms of service
+            </FooterLink>
+            <Delimiter />
+            <FooterLink internal url="/privacy">
+              Privacy policy
+            </FooterLink>
+          </div>
+          {!md && (
+            <>
```

Comment:
Hm, probably we can remove this `<></>`, because only one `<div>...</div>` is returned here